highlight.io/components/Home/CalendlyPopover.tsx

```diff
@@ -1,10 +1,23 @@
 import { Popover } from '@headlessui/react'
 import { ArrowRightCircleIcon, XMarkIcon } from '@heroicons/react/20/solid'
 import classNames from 'classnames'
+import { useRouter } from 'next/router'
+import { useEffect } from 'react'
 import { InlineWidget } from 'react-calendly'
 import { Typography } from '../common/Typography/Typography'
 
 export const CalendlyPopover = () => {
+	const router = useRouter()
+
+	useEffect(() => {
+		window.addEventListener('message', (message) => {
+			const { event } = message.data
+
+			if (event === 'calendly.event_scheduled')
```

Comment:
Do you need to do this? I think when the event is scheduled, it already redirects to this page 🤔 .

Reply:
I tested it with calendly's test widget and it doesn't seem to redirect. They fire a window event after the schedule is done